### PR TITLE
Minor editorial issue: ref to stream section of RFC9000

### PR DESCRIPTION
**Pull request by mirjak**
Thanks for the well written document! I thought this direct pointer to the right section might help!

### CODE REVIEW

File: draft-ietf-dprive-dnsoquic.md [hunk 1]
```diff
@@ -296,9 +296,9 @@ The server MUST send the response(s) on the same stream and MUST indicate, after
 the last response, through the STREAM FIN mechanism that no further data will be
 sent on that stream.
 
-Therefore, a single client-initiated DNS transaction consumes a single stream.
+Therefore, a single unidirectional client-initiated DNS transaction consumes a single stream.
```
Comment (saradickinson) [inline draft-ietf-dprive-dnsoquic.md:299]: I think the reference below is useful - thank you! However the streams used by DoQ are bidirectional (as are DNS transactions) so I don't think `unidirectional` here is correct unless I missed your point?

Comment (mirjak) [reply draft-ietf-dprive-dnsoquic.md:299]: Sorry, you are right. This should be bidirectional. In QUIC there are both unidirectional and bidirectional stream and therefore I thought it's useful to clarify. however, this is probably anyway in the wrong position. Let me try another way...

Comment (mirjak) [inline draft-ietf-dprive-dnsoquic.md:299]:
```suggestion
Therefore, a single DNS transaction consumes a single unidirectional client-initiated stream.
```
This should be correct now.

Comment (martinthomson) [reply draft-ietf-dprive-dnsoquic.md:299]: I think that you mean:

```suggestion
Therefore, a single DNS transaction consumes a single bidirectional client-initiated stream.
```

Comment (huitema) [reply draft-ietf-dprive-dnsoquic.md:299]: +1 @martinthomson

**Review by huitema — APPROVED** [review]
With the replacement of unidir by bidir, this is good.

**Review by saradickinson — APPROVED** [review]

**Pull request merged**